### DISCUSSION TITLE
Problem with the initialization using OMP installed via Homebrew

### DISCUSSION THREAD

**Discussion by lewis-yeung** [JanDeDobbeleer/oh-my-posh#2644] [General]
For the time being, there is a problem with the initialization using Oh My Posh installed via Homebrew:

As we know, after an upgrade via Homebrew, the absolute path of the latest OMP executable will change due to the change of the version number. The absolute path is used in the initialization script, however the old path will become invalid after an upgrade, so it cannot render a prompt any more in the current shell. For other shells we just need to restart the shell to get it work again, while for Nushell this can be quite annoying since we have to run `oh-my-posh init nu ...` manually to recreate a new script.

**Comment by JanDeDobbeleer** [comment]
@lewis-yeung I'm using nu shell fulltime, how come I've not seen that? There's always the `--strict` parameter too which uses the executable name and not the path.

**Reply by lewis-yeung** [reply]
Simple and effective solution! I think we can mention this in docs.
